pkg/api/types.go

```diff
@@ -370,6 +370,9 @@ type PersistentVolumeSource struct {
 	// ScaleIO represents a ScaleIO persistent volume attached and mounted on Kubernetes nodes.
 	// +optional
 	ScaleIO *ScaleIOVolumeSource
+	// LocalVolume represents a local volume accessible only from one node
+	// +optional
+	LocalVolume *LocalVolumeSource
```

Comment:
Why have the word "volume" in the name?  Is that intentional or just wasn't considered?  Why not `Local` ?  This is a question, not a request for change (yet)